provisioning/roles/docker/files/docker-compose.yml

```diff
@@ -0,0 +1,171 @@
+version: "3"
+
+services:
+  elasticsearch:
+    image: elasticsearch:5.6.10
+    container_name: elasticsearch
+    restart: always
+    environment:
+      - "bootstrap.memory_lock=true"
```

Comment:
what does this do? can we add a comment

Reply:
It is explained [here](https://www.elastic.co/guide/en/elasticsearch/reference/5.6/setup-configuration-memory.html#mlockall). I'll add a short comment.